conftest.py

```
@@ -0,0 +1,236 @@
+@pytest.fixture(scope="session")
+def tensorized_MEDS_dataset(simple_static_MEDS: Path) -> Path:
+    """Override upstream fixture so ``MTD_preprocess`` (and its children) resolve from the active venv."""
+    import os
+
+    env = os.environ.copy()
+    env["PATH"] = _VENV_BIN + os.pathsep + env.get("PATH", "")
+
+    with tempfile.TemporaryDirectory() as cohort_dir:
+        cohort_dir = Path(cohort_dir)
+        cmd = f"MTD_preprocess MEDS_dataset_dir={simple_static_MEDS!s} output_dir={cohort_dir!s}"
+        out = subprocess.run(cmd, shell=True, check=False, capture_output=True, env=env)
```

Ruff (0.15.7)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)